php/libraries/Utility.class.inc

```diff
@@ -283,6 +283,118 @@ class Utility
         return self::getSubprojectList($projectID);
     }
 
+    /**
```

Comment:
I don't think these functions are supposed to be added in this PR. It seems to be the result of a bad rebase since the functions were moved.. 

This PR should be re-based onto master, and hopefully this change will disappear. If not the functions need to be deleted from here. After rebasing, the functions need to be tested in their new location.

Reply:
@driusan they were previously deleted, i redeleted them. lets see if travis is going to complain